src/Config/Environment.php

```diff
@@ -34,7 +34,7 @@ class Environment
     public function __construct($homeDir, $cwd, $autoloadFile)
     {
         $this->homeDir = $homeDir;
-        $this->originalCwd = Path::canonicalize($cwd);
+        $this->originalCwd = FsUtils::realpath($cwd);
```

Comment:
`FsUtils::realpath` is a lot safer than a raw `realpath` call, so perhaps this is okay. If we add this, it means that we'll have to work harder in the AppVeyor tests, because this will convert Windows paths from using `/` to `\`.

This only introduces a handful of failures in the current tests, so I suppose it is tractable.

Reply:
@greg-1-anderson, I'm not sure what the right way is to fix the tests. Can you point me in the right direction? Thanks!

Reply:
Our typical strategy here is to normalize the output before asserting. For example, if you str_replace each `\` with a `/`, then you can compare the path invariantly again.

Reply:
Makes sense. Let's see if this fixes the tests.